### PR TITLE
cmake 3.2.2

### PR DESCRIPTION
**Pull request by andystanton**

### CODE REVIEW

File: Library/Formula/cmake.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Cmake < Formula
   homepage "http://www.cmake.org/"
-  url "http://www.cmake.org/files/v3.2/cmake-3.2.1.tar.gz"
-  sha1 "53c1fe2aaae3b2042c0fe5de177f73ef6f7b267f"
+  url "http://www.cmake.org/files/v3.2/cmake-3.2.2.tar.gz"
+  sha1 "d22be01828508027617494c8b59d5999d4c1490e"
```
Comment (DomT4) [inline Library/Formula/cmake.rb]: `sha256` would be good, if you're feeling particularly kind :smiley_cat:

**Comment by andystanton** [comment]
@DomT4 added sha256 and squashed to single commit.

**Comment by DomT4** [comment]
@andystanton Thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**